BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -359,7 +359,8 @@ - (NSString *)fullFolderFieldsParameterString
                        BOXAPIObjectKeyAllowedInviteeRoles,
                        BOXAPIObjectKeyAllowedSharedLinkAccessLevels,
                        BOXAPIObjectKeyCollections,
-                       BOXAPIObjectKeyCollectionMemberships];
+                       BOXAPIObjectKeyCollectionMemberships,
+                       BOXAPIObjectKeyClassification];
```

Comment:
How do people feel about this being in the full fields for all SDK consumers?

Reply:
I think one of the main motivations to put this field here was so that in later stages when we need the information in other places besides the Files Info page, we won't need to re-retrieve the information again. The results of the investigation also showed that the including this field have an significant impact on the response timing (0.68 seconds increase for folder with 100 items on WiFi and less on 3G). But, would love to hear everyone else's thoughts.